Bottom: 6128bb07cdb22f2cf61e90645eb265d3232ef90b
Top:    42da0e4aba13665db7b6f7aebb15187b85cd54ed
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2009-11-06 17:00:51 +0000

IMPORTANT: DONT DRAW POUR OUTLINES DURING EXPORT

They can produce shorted boards otherwise!


---

diff --git a/src/draw.c b/src/draw.c
index 49f011f..1c1a504 100644
--- a/src/draw.c
+++ b/src/draw.c
@@ -708,7 +708,8 @@ pour_callback (const BoxType * b, void *cl)
   struct pin_info *i = (struct pin_info *) cl;
   PourType *pour = (PourType *)b;
 
-  DrawPour (i->Layer, pour, 0);
+  if (gui->gui)
+    DrawPour (i->Layer, pour, 0);
 
   if (pour->PolygonN)
     {
